README.md

```diff
@@ -1 +1,72 @@
-# Capstone-project-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# restuarant-menu
+
+> This website is about how to order food online and also satisfy your cravings. we deliver at your door step.
+
+
+## Built With
+
+- HTML/CSS3 
+- Javascript
+- Linters (Webhint, Eslint, Stylelint)
+- Visual Studio
+
+## Live Demo (if available)
+
+[Live Demo Link](https://rmjspecial.github.io/Capstone-project-1/)
+[Video Link](https://www.loom.com/share/9ad514e15fb44746a69df0271a99e3ea)
```

Comment:
- [ ] Please, consider that the demo live link is still not working
- [ ] Please, consider recording the new video with, suggested by the previous reviewer, the part of code you proud of 👍

Reply:
@elmar8287 the live demo is not working because I have not pushed it to my main they are all in the branch and I created it from the main.

Reply:
the previous code reviewer asked me to do a video showing my code because in the previous video I did my code was not displayed maybe you can throw more light on what you meant by this thanks.